types/context.go

```diff
@@ -23,43 +23,47 @@ but please do not over-use it. We try to keep all data structured
+	voteInfo             []abci.VoteInfo
+	gasMeter             GasMeter
+	blockGasMeter        GasMeter
+	checkTx              bool
+	recheckTx            bool // if recheckTx == true, then checkTx must also be true
+	minGasPrice          DecCoins
+	consParams           *tmproto.ConsensusParams
+	eventManager         *EventManager
+	priority             int64 // The tx priority, only relevant in CheckTx
+	kvGasConfig          storetypes.GasConfig
+	transientKVGasConfig storetypes.GasConfig
 }
 
 // Proposed rename, not done to avoid API breakage
 type Request = Context
 
 // Read-only accessors
-func (c Context) Context() context.Context    { return c.baseCtx }
-func (c Context) MultiStore() MultiStore      { return c.ms }
-func (c Context) BlockHeight() int64          { return c.header.Height }
-func (c Context) BlockTime() time.Time        { return c.header.Time }
-func (c Context) ChainID() string             { return c.chainID }
-func (c Context) TxBytes() []byte             { return c.txBytes }
-func (c Context) Logger() log.Logger          { return c.logger }
-func (c Context) VoteInfos() []abci.VoteInfo  { return c.voteInfo }
-func (c Context) GasMeter() GasMeter          { return c.gasMeter }
-func (c Context) BlockGasMeter() GasMeter     { return c.blockGasMeter }
-func (c Context) IsCheckTx() bool             { return c.checkTx }
-func (c Context) IsReCheckTx() bool           { return c.recheckTx }
-func (c Context) MinGasPrices() DecCoins      { return c.minGasPrice }
-func (c Context) EventManager() *EventManager { return c.eventManager }
-func (c Context) Priority() int64             { return c.priority }
+func (c Context) Context() context.Context                   { return c.baseCtx }
+func (c Context) MultiStore() MultiStore                     { return c.ms }
+func (c Context) BlockHeight() int64                         { return c.header.Height }
+func (c Context) BlockTime() time.Time                       { return c.header.Time }
+func (c Context) ChainID() string                            { return c.chainID }
+func (c Context) TxBytes() []byte                            { return c.txBytes }
+func (c Context) Logger() log.Logger                         { return c.logger }
+func (c Context) VoteInfos() []abci.VoteInfo                 { return c.voteInfo }
+func (c Context) GasMeter() GasMeter                         { return c.gasMeter }
+func (c Context) BlockGasMeter() GasMeter                    { return c.blockGasMeter }
+func (c Context) IsCheckTx() bool                            { return c.checkTx }
+func (c Context) IsReCheckTx() bool                          { return c.recheckTx }
+func (c Context) MinGasPrices() DecCoins                     { return c.minGasPrice }
+func (c Context) EventManager() *EventManager                { return c.eventManager }
+func (c Context) Priority() int64                            { return c.priority }
+func (c Context) KVGasConfig() storetypes.GasConfig          { return c.kvGasConfig }
```

Comment:
Can we update this too?
https://github.com/cosmos/cosmos-sdk/blob/ff4f995b185f0b7e318bd0a47d0b1aa824406d8b/x/gov/keeper/msg_server.go#L56

Reply:
Updated